config.properties.example

```diff
@@ -4,6 +4,9 @@ log_level=info
 producer=kafka
 kafka.bootstrap.servers=localhost:9092
 
+Set Timeout for connect to binary client, default is 5000ms
+#timeout_binary_log_client=50000
```

Comment:
To be exact, the timeout is used to connect to MySQL server to establish replication stream. I'm thinking `replication_connection_timeout` might be clearer.